test/runtests.jl

```diff
@@ -1,5 +1,6 @@
 using Test
 using LinearFractional
+using LinearAlgebra
```

Comment:
Is this change part of the same issue, or just something you noticed was needed?

Reply:
With Julia 1.0, we need to import `LinearAlgebra` to access linear algebra functions like `dot`.  We just didn't access any linear algebra in the tests prior to this new test.